src/main/webapp/creatives/maps/interaction.js

```diff
@@ -0,0 +1,45 @@
+
+import {generateSessionId} from '../maps/utils.js';
+
+const URL = {
+  INITIALIZE: "initialize",
+  FIND_NEAREST_LOCATION: "find-location",
+  GRANT_LOCATION_DATA: "grant-location",
+  MAP_INTERACT: "map-interact",
+  SKIP_TO_CONTENT: "skip-to-content",
+  RETURN_TO_AD: "return-to-ad"
+}
+
+class User {
+  constructor() {
+    this.correlator = generateSessionId();
+  }
+
+  sendRequest_(url) {
+    fetch(`/${url}?correlator="${this.correlator}"`).then(() => console.log("success"));
```

Comment:
console.log("success") isn't a useful message.  Ideally change to a useful message or remove.